From 9a6006f88c844562c7a7c286a5b000cb946ce026 Mon Sep 17 00:00:00 2001
From: Li Jun <jun.li@nxp.com>
Date: Wed, 13 Jan 2016 11:53:31 +0800
Subject: [PATCH 0850/5242] MLK-12248-1 ARM: imx6qdl-sabresd: add tx-d-cal
 property for usbphy

commit  79ca4444815f5cd77ce45220aba57139793e8522 from
https://source.codeaurora.org/external/imx/linux-imx.git

Set tx-d-cal to be 0x5 to improve usb signal quality for all imx6qdl
sabresd boards.

Signed-off-by: Li Jun <jun.li@nxp.com>
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 arch/arm/boot/dts/imx6qdl-sabresd.dtsi |    8 ++++++++
 1 file changed, 8 insertions(+)

diff --git a/arch/arm/boot/dts/imx6qdl-sabresd.dtsi b/arch/arm/boot/dts/imx6qdl-sabresd.dtsi
index 2b8a4c8..c7dd824 100644
--- a/arch/arm/boot/dts/imx6qdl-sabresd.dtsi
+++ b/arch/arm/boot/dts/imx6qdl-sabresd.dtsi
@@ -1109,6 +1109,14 @@
 	status = "okay";
 };
 
+&usbphy1 {
+	tx-d-cal = <0x5>;
+};
+
+&usbphy2 {
+	tx-d-cal = <0x5>;
+};
+
 &usdhc2 {
 	pinctrl-names = "default";
 	pinctrl-0 = <&pinctrl_usdhc2>;
-- 
1.7.9.5

